File: ArraysAndStrings/1.1 - IsUnique/IsUnique.py
def isUnique(word):
    unique = True
    for x in word:
        count = 0
        for y in word:
            if x == y:
                count = count + 1
                if count > 1: 
                    print(word + " is not unique.")
                    return False
    print (word + " is unique.")
    return True
    
isUnique("step")
isUnique("stepe")

File: ArraysAndStrings/1.1 - IsUnique/isUnique.java
class isUnique{
    static boolean isUnique(String word){ 
        for(int i=0; i<word.length(); i++){
            int count = 0;
            for(int j=0; j<word.length(); j++){
                if(word.charAt(i) == word.charAt(j)){
                    count++;
                    if(count > 1){
                        System.out.println(word + " is not unique.");
                        return false;
                    }
                }
            }
        }
        System.out.println(word + " is unique.");
        return true;
    }
    public static void main(String args[]) {
        isUnique("Hello");       
        isUnique("Dog");
    }
}